CondFormats/BTauObjects/src/BTagEntry.cc

```diff
@@ -17,8 +22,8 @@ BTagEntry::Parameters::Parameters(
   float discr_max
 ):
   operatingPoint(op),
-  measurementType(measurement_type),
-  sysType(sys_type),
+  measurementType(makeLowerCase(measurement_type)),
+  sysType(makeLowerCase(sys_type)),
```

Comment:
I've made it static and used references. But I had to retain the return statement in order to use it in the constructor of BTagEntry::Parameters.